.circleci/config.yml

```diff
@@ -137,7 +137,7 @@ jobs:
           name: Run tests with coverage
           command: |
             source env/bin/activate
-            coverage run --source cla_backend manage.py test --verbosity=2
+            coverage run --source cla_backend --omit='*migrations*' manage.py  test --verbosity=2
```

Comment:
🙋‍♂ Might be worth putting these ignores and the default values to [configuration files](https://coverage.readthedocs.io/en/v4.5.x/config.html#config), so locally we can reproduce the same results without repeating these exact arguments.